src/highlight.js

```diff
@@ -238,6 +241,42 @@ https://highlightjs.org/
     }
   }
 
+  function commonKeyword(word) {
+    return COMMON_KEYWORDS.indexOf(word.toLowerCase()) != -1
+  }
+
+  function scoreForKeyword(keyword, providedScore) {
+    // manual scores always win over common keywords
+    // so you can force a score of 1 if you really insist
+    if (providedScore)
+      return Number(providedScore)
+
+    return commonKeyword(keyword) ? 0 : 1;
+  }
+
+  function compileKeywords(rawKeywords, case_insensitive) {
+      var compiled_keywords = {};
+
+      var flatten = function(className, str) {
+        if (case_insensitive) {
+          str = str.toLowerCase();
+        }
+        str.split(' ').forEach(function(kw) {
+          var pair = kw.split('|');
+          compiled_keywords[pair[0]] = [className, scoreForKeyword(pair[0], pair[1])];
+        });
+      };
+
+      if (typeof rawKeywords === 'string') { // string
+        flatten('keyword', rawKeywords);
+      } else {
+        objectKeys(rawKeywords).forEach(function (className) {
```

Comment:
No, idea, it was that way and I left it that way for now.  When I'm new to a code base I start small... and one time make bigger and bigger changes, see what people are open to, what they aren't etc...  otherwise you can very quickly create a "I'm the boss now and we're doing things my way vibe."... and that usually goes poorly.

But I totally agree this seems pointless. :)